README.md

```diff
@@ -207,25 +226,33 @@ Debug information about the respooler system can be found by visiting the follow
 
 `{ip address}/printer/objects/query?AFC`
 
-### LEDs not displaying correct color
+<details>
+<summary>LEDs not displaying correct color</summary>
```

Comment:
Summary sections look weird and don't apply then code sections and new lines when looking at the rich text
![image](https://github.com/user-attachments/assets/0477741a-c956-4308-a237-aa04262fbafd)

Reply:
Gah. They showed up fine for me. I'll revert them.

Reply:
@jimmyjon711 Adjusted.

Reply:
Looks good to me approved